var/spack/repos/builtin/packages/intel-oneapi-runtime/package.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+import os
+
+from llnl.util import tty
+
+from spack.package import *
+from spack.pkg.builtin.gcc_runtime import get_elf_libraries
+
+
+class IntelOneapiRuntime(Package):
+    """Package for OneAPI compiler runtime libraries"""
+
+    homepage = "https://software.intel.com/content/www/us/en/develop/tools/oneapi.html"
+    has_code = False
+
+    tags = ["runtime"]
+
+    requires("%oneapi")
+
+    depends_on("gcc-runtime", type="link")
+
+    LIBRARIES = [
+        "imf",
+        "intlc",
+        "irng",
+        "svml",
+        "ifcore",  # Fortran
+        "ifcoremt",  # Fortran
+        "ifport",  # Fortran
+        "sycl",
```

Comment:
What about libiomp5?

I have a problem now where mkl wants to depend on a virtual openmp runtime that could be provided by gcc or icx: https://github.com/spack/spack/issues/42310